src/ansys/dpf/core/operators/serialization/hdf5dpf_generate_result_file.py

```diff
@@ -185,10 +187,12 @@ def _spec():
                 ),
                 3: PinSpecification(
                     name="ansys_unit_system_id",
-                    type_names=["int32"],
+                    type_names=["int32", "result_info"],
                     optional=True,
-                    document="""Defines the unitsystem the results are
-        exported with.""",
+                    document="""Defines the unit system the results are
+        exported with. a result info can be
```

Comment:
```suggestion
        exported with. A result info can be
```

Reply:
Why are we using the `ansys_unit_system_id` pin to provide a ResultInfo and export `analysis_type` and `physics_type`?

Reply:
@PProfizi To be backwards compatible... At a certain point (when we have a mechanism to deprecate), ansys_unit_system_id should not be supported, but the result_info